src/Samples/PortableTask/PortableTask.csproj

```diff
@@ -12,7 +12,12 @@
   </ItemGroup>
   <ItemGroup>
     <PackageReference Include="Microsoft.Build.Framework" Version="15.5.180" />
-    <PackageReference Include="Microsoft.Build.Utilities.Core" Version="15.5.180" />
+    <PackageReference Include="Microsoft.Build.Utilities.Core" Version="15.5.180" GeneratePathProperty="true" />
   </ItemGroup>
   <Target Name="UpdateXlf" />
+
+  <!-- This is only needed for a test in the MSBuild repo; it is unrelated to the PortableTask sample itself. -->
+  <Target Name="CopyMSBuildUtilitiesToNewFolder" AfterTargets="Build">
```

Comment:
I'd expected Restore to work, but it didn't. In lieu of a suggestion, Build works.

Reply:
Since this is in our Samples folder, I would really like it to be a good example. Hooking `Build` is a bad practice because while it usually works ok, it can break in extremely hard-to-debug ways when referenced by a project that calls different targets.

Since this is copying a file to the output directory, how about `BeforeTargets="CopyFilesToOutputDirectory"`?